harmonic/install_windows_src.md

```diff
@@ -143,7 +143,11 @@ This is the end of the source install instructions; head back to the [Getting st
 page to start using Gazebo!
 
 > **NOTE**
-> As Gazebo GUI is not yet working, running `gz sim` will not work. You can run only the server with
+> To launch Gazebo GUI , run 
+>```bat
+>gz sim
```

Comment:
I do not think this works (see https://github.com/gazebosim/gz-sim/issues/2393). What is currently running is `gz sim -s` and `gz sim -g` separately, a bit like on macOS, I guess we can take the macOS documentation as example?

Reply:
Yes My mistake , corrected it